test/mongoose_push_web/test_helper.exs

```diff
@@ -0,0 +1,2 @@
+{:ok, _} = Application.ensure_all_started(:mongoose_push)
+ExUnit.start()
```

Comment:
Missing end-of-line character.

Reply:
Also, it's recommended to use `ExUnit.start(capture: true)` in order to hide logs as long tests are green.